src/Chapter2.hs

```diff
@@ -621,7 +681,15 @@ Write a function that takes elements of a list only in even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [a] -> [a]
+takeEven l = go (length l) l
+  where
+    go :: Int -> [a] -> [a]
+    go 0 _ = []
+    go 1 n = n
+    go 2 (x:_:xys) = x : xys
+    go i (x:_:xys) = x : (go (i-2) xys)
```

Comment:
Very creative! 🎨 

Note that you can implement this function with simple pattern matching as well. It requires fewer function calls and a bit more efficient by memory and time as you use `length`, which is not optimal.